src/Core/AdminConsole/OrganizationFeatures/Policies/PolicyRequirements/OrganizationDataOwnershipPolicyRequirement.cs

```diff
@@ -72,6 +72,21 @@ public bool RequiresDefaultCollectionOnConfirm(Guid organizationId)
     {
         return _policyDetails.Any(p => p.OrganizationId == organizationId);
     }
+
+    /// <summary>
+    /// Determines if a user is eligible for self-revocation under the Organization Data Ownership policy.
```

Comment:
Worth noting something like: "Self-revoke is used to opt out of migrating the user's personal vault to the organization pursuant to this policy." Just to explain how self-revoke relates to this policy.